batch/batch/constants.py

```diff
@@ -1 +1,3 @@
 ROOT_JOB_GROUP_ID = 0
```

Comment:
The ROOT_JOB_GROUP_ID should be defined in exactly one place, since we need it both client-side and server-side, that place has to be hailtop.